pkg/peer/webrtc_callbacks.go

```diff
@@ -11,44 +10,33 @@ import (
 // A callback that is called once we receive first RTP packets from a track, i.e.
 // we call this function each time a new track is received.
 func (p *Peer[ID]) onRtpTrackReceived(remoteTrack *webrtc.TrackRemote, receiver *webrtc.RTPReceiver) {
-	// Create a local track, all our SFU clients that are subscribed to this
-	// peer (publisher) wil be fed via this track.
-	localTrack, err := webrtc.NewTrackLocalStaticRTP(
-		remoteTrack.Codec().RTPCodecCapability,
-		remoteTrack.ID(),
-		remoteTrack.StreamID(),
-	)
+	// Get the simulcast layer from RID.
+	layer, err := RIDToSimulcastLayer(remoteTrack.RID())
 	if err != nil {
-		p.logger.WithError(err).Error("failed to create local track")
+		p.logger.WithError(err).Error("failed to parse RID")
```

Comment:
We should handle non-simulcast tracks too, right?

Reply:
Yep, I've added this behavior in the upcoming commits (perhaps something went wrong with the rebase), but you probably have already noticed it in the newer commits :)